li-utils/src/main/pegasus/com/linkedin/common/DatasetFieldUrn.pdl

```diff
@@ -0,0 +1,28 @@
+namespace com.linkedin.common
+
+/**
+ * Standardized dataset field information identifier.
+ */
+@java.class = "com.linkedin.common.urn.DatasetFieldUrn"
+@validate.`com.linkedin.common.validator.TypedUrnValidator` = {
+  "accessible" : true,
+  "owningTeam" : "urn:li:internalTeam:datahub",
+  "entityType" : "datasetField",
+  "constructable" : true,
+  "namespace" : "li",
+  "name" : "DatasetField",
+  "doc" : "Standardized dataset field information identifier",
+  "owners" : [ "urn:li:corpuser:fbar", "urn:li:corpuser:bfoo" ],
+  "fields" : [ {
+    "type" : "com.linkedin.common.urn.DatasetUrn",
+    "name" : "dataset",
+    "doc" : "Dataset that this dataset field belongs to."
+  }, {
+    "name" : "field",
+    "doc" : "Dataset field path or column name",
```

Comment:
I think we should just standardize it to field path even for non-nested fields.

Reply:
Sure. Let us standardize to fieldPath.